src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  let price = 0;
```

Comment:
It's better to use a `const` for the base rental cost (40) and discounts (20, 50) to improve code readability and maintainability. 😊

Reply:
Code is rewrited